cypress/integration/products_page.spec.js

```diff
@@ -34,69 +39,51 @@ describe('My First Test', () => {
       cy.get('[id="addToBasketButton-1"]')
         .click();
   
-      cy.get('[id="basketTotal"]').should('not.contain', '0');
-      cy.get('[id="basketLength"]').should('not.contain', '0');
+      cy.get('[id="basketTotal"]')
+        .should('not.contain', '0');
+      cy.get('[id="basketLength"]') 
+        .should('contain','2'); 
     }); 
+
+    it('should clean the basket', () => {
+      cy.get('[id="clearTheBasketButton"]')
+        .click();
+  
+      cy.get('[id="basketTotal"]')
+        .should('contain', '0');
+      cy.get('[id="basketLength"]')
+        .should('contain', '0');
+    });
   
     it('should open quickView', () => {
       cy.get('[id="quickViewButton"]')
         .click();
   
-      cy.get('.quickviewContainer').should('be.visible');
+      cy.get('.quickviewContainer')
+        .should('be.visible');
     });
       
     it('should close quickView', () => {
       cy.get('[id="quickViewButton"]')
         .click();
   
-      cy.get('.quickviewContainer').should('not.be.visible');  
+      cy.get('.quickviewContainer')
+        .should('not.be.visible');  
     }); 
   
-    it('should clean the basket', () => {
-      cy.get('[id="clearTheBasketButton"]')
-        .click();
-  
-      cy.get('[id="basketTotal"]').should('contain', '0');
-      cy.get('[id="basketLength"]').should('contain', '0');
-    });
-
-    it('should check if product is visible in quickview', () => {
-      cy.get('[id="quickViewButton"]')
-        .click();
-      cy.get('[id="addToBasketButton-1"]')
-        .click();   
-
-      cy.get('.quickviewProduct')
-        .should('be.visible');  
-    });
-
-    it('should check if product can be removed from quickview', () => {
-      cy.get('.quickviewProduct')
-        .find('.closeIcon')
-        .click();
-      
-      cy.get('quickviewProduct')
-        .should('not.be.visible');  
-      cy.get('[id="basketTotal"]').should('contain', '0');
-      cy.get('[id="basketLength"]').should('contain', '0');
-      cy.get('[id="quickViewButton"]')
-        .click();
-    });
-
-    it('should check if many products can be added to quickview', () => {
+    it('should check if user can add every products to the basket', () => {
       cy.get('[id="addToBasketButton-0"]')
         .click();
       cy.get('[id="addToBasketButton-1"]')
         .click();  
       cy.get('[id="addToBasketButton-2"]')
         .click();      
-      cy.get('[id="quickViewButton"]')
-        .click();    
-
+      cy.get('[id="addToBasketButton-3"]')
+        .click();   
+        
       cy.get('.quickviewContainer')
```

Comment:
This tests failing because i couldn't click on the button. I still can't.  I understood from FB conversation that this is no problem for now, so I removed test. Please, tell me is that ok for you